### PR TITLE
KI Tagging in KIM

### PR DESCRIPTION
**Pull request by bpneal**

### CODE REVIEW

**Comment by jane-doe** [comment]
Can one of the admins verify this patch?

File: NTO/Automation/entities/KnowledgeItem.ttl [hunk 1]
```diff
@@ -76,5 +76,6 @@ In that case, the KnowledgeItem executes operations, which serves the exploitati
 			[ ogit:connects  ogit.Automation:KnowledgeItem ]
 			[ ogit:connects  ogit.Forum:Post ]
 			[ ogit:connects  ogit:Hyperlink ]
+			[ ogit:contains  ogit:Tag ]
```
Comment (cy303) [inline NTO/Automation/entities/KnowledgeItem.ttl:79]: I would question semantics here... a KnowledgeItem does not "contain" a tag but a Tag "describes" or "qualifies" a KnowledgeItem, doesn't it?

File: SGO/sgo/entities/Tag.ttl [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	rdfs:subClassOf ogit:Entity;
+	rdfs:label "Tag";
+	dcterms:description "An Tag is a customer defined tag to help filter data";
+	# For ranges, see http://dublincore.org/documents/dcmi-period/
+	dcterms:valid "start=2020-08-04;";
+	dcterms:creator "Ben Neal";
+	ogit:scope "SGO";
+	ogit:parent ogit:Node;
+	ogit:mandatory-attributes (
+		ogit:name
```
Comment (cy303) [inline SGO/sgo/entities/Tag.ttl:16]: If it's really just the name, nothing would keep me from adding many tags with same name here, right?

File: NTO/Automation/entities/KnowledgeItem.ttl (hunk 1 above)
Comment (bpneal) [reply NTO/Automation/entities/KnowledgeItem.ttl:79]: Fair point, I'll change this

Comment (bpneal) [reply NTO/Automation/entities/KnowledgeItem.ttl]: I have removed the relationship from the Knowledge Item and have included `ogit:Tag ogit:describes ogit:Automation:KnowledgeItem`

File: SGO/sgo/entities/Tag.ttl (hunk 2 above)
Comment (bpneal) [reply SGO/sgo/entities/Tag.ttl:17]: Name shouldn't be unique as can be reused across instances, and even re-used on the same instance. I don't see a use-case where we'd need the name to be unique.
I have also added an option attribute `color`

File: NTO/Automation/entities/KnowledgeItem.ttl (hunk 1 above)
Comment (bpneal) [reply NTO/Automation/entities/KnowledgeItem.ttl]: I have removed the edge from tag altogether now

**Pull request merged**